### ISSUE TITLE
TransferTransaction throw protocol error but completes anyway

### ISSUE THREAD

**Issue by ericleponner** [hashgraph/hedera-sdk-swift#37]
### Description

I run a `TransferTransaction` on `mainnet` which thrown the following error:

`grpc: status: Internal, message: "protocol error: received message with invalid compression flag: 60 (valid flags are 0 and 1) while receiving response with status: 503 Service Unavailable", details: [], metadata: MetadataMap { headers: {"content-length": "107" "cache-control": "no-cache" "content-type": "text/htm|"} }`

However transaction completed successfully on `mainnet` (`status == .success`)

### Steps to reproduce

Here is the code I used:

```
                // 1) Executes transaction
                let client = controller.model.network.makeClient()
                client.setOperator(payerAccountId!, payerKey!)
                response = try await TransferTransaction()
                      .hbarTransfer(sourceAccount, -tbarAmount)
                      .hbarTransfer(targetAccountId!, +tbarAmount)
                      .execute(client)
               
                // 2) Waits for receipt
                receipt = try await response!.getReceipt(client)

```                

### Hedera network

mainnet

### Version

v0.13.0

### Operating system

macOS

**Comment by izik1** [comment]
Hi, spent some time writing a more minimal example today (namely, this one is free)

```swift
        let client = try Client.forMainnet()

        let results = try await AccountBalanceQuery().accountId(3).execute(client)

        print(results)
```

Executed it 40 times, triggered once... Yikes.

**Comment by izik1** [comment]
Appears to trigger specifically on the node with account id `0.0.7`, lemme check in with someone.

**Comment by izik1** [comment]
This should be fixed as of the latest release (0.14), if it isn't feel free to re-open the issue.
